vllm/compilation/pass_manager.py

```diff
@@ -132,4 +137,13 @@ def uuid(self):
             state["passes"].append(pass_.uuid())
         state["passes"].append(self.fix_functionalization.uuid())
 
+        # If there is no pass context, we are likely re-using compilation config
+        # In this case uuid() is called not at compilation time,
+        # we don't have the compile range information.
+        pass_context = try_get_pass_context()
```

Comment:
@ProExpertProg Do you think we need to propagate the vllm config here so that we could have at least the default range? I believe the best option would be to fix the issue with non-set pass_context in case of compilation config but probably not in this PR.